service/src/intTest/java/uk/nhs/adaptors/gp2gp/MessageQueueTest.java

```diff
@@ -31,6 +56,36 @@ public void When_SendingInvalidMessage_Expect_MessageIsSentToDeadLetterQueue() t
         jmsTemplate.send(inboundQueueName, session -> session.createTextMessage(INVALID_MESSAGE_CONTENT));
         Message message = jmsTemplate.receive(DLQ);
         assert message != null;
-        assertThat(message.getBody(String.class)).isEqualTo(INVALID_MESSAGE_CONTENT);
+        assertThat(message.getBody(String.class), is(INVALID_MESSAGE_CONTENT));
+    }
+
+    @Test
+    public void When_SendingValidMessage_Expect_MessageIsAddedToDB() throws IOException, InterruptedException {
```

Comment:
Do this as a component test in src/test without SpringBootTest.

Reply:
Do you mean changing it to unit test which directly tests EhrStatusConsumerService?